interface/interface.dm

```diff
@@ -66,6 +66,18 @@
 		to_chat(src, "<span class='danger'>The rules URL is not set in the server configuration.</span>")
 	return
 
+/client/verb/github()
+	set name = "GitHub"
+	set desc = "Visit the GitHub page."
+	set hidden = 1
+	if(config.githuburl)
+		if(alert("This will open our source code page on GitHub. Are you sure?",,"Yes","No")=="No")
```

Comment:
I'd probably change this to actually be like "open our repository" or something. Or even "Github page", since this technically is more than just our source code.

But semantics aside, this is a nice little addition.